canonical-web-and-design/maas-ui
# refactor(ui): update fabric to use a slice

## Done

- Replaced the fabric reducers and actions with a slice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to a KVM details and open the Compose form
- Add an interface and open the subnet select
- Check that the subnets/fabrics are correctly grouped by space

## Fixes

Fixes #1746
